auth-service/src/main/java/com/piggymetrics/auth/controller/UserController.java

```diff
@@ -4,10 +4,7 @@
 import com.piggymetrics.auth.service.UserService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.security.access.prepost.PreAuthorize;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
its's a bad practice to use asterisk imports, please
if you use intellij: https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports